packages/components/form-field/stepper.ts

```diff
@@ -61,6 +65,15 @@ export class KbqStepper {
     /** Emitted when the stepper is decremented. */
     @Output() readonly stepDown: EventEmitter<void> = new EventEmitter<void>();
 
+    /** @docs-private */
+    protected readonly mouseUp = new Subject<void>();
+
+    /** @docs-private */
+    protected readonly mouseHoldInterval = timer(300).pipe(
```

Comment:
мб так и обозвать переменную?
```suggestion
    protected readonly longPress = timer(300).pipe(
```

---

давай добавим константы для 300 и 75 (в том числе для использования в тестах)

Reply:
добавил